Sources/tart/Logging/Logger.swift

```diff
@@ -12,16 +12,13 @@ var defaultLogger: Logger = {
     return InteractiveConsoleLogger()
   }
 }()
+var jsonLogger: Logger = NopLogger()
```

Comment:
Why it's a `NopLogger`?

Reply:
Because we don't output JSON by default.